spec/gnarails/cli/application_spec.rb

```diff
@@ -12,7 +12,7 @@ module Cli
           application.options = { webpack: "react", skip_yarn: false, skip_git: true }
           application.new("name")
 
-          default_options = "-m #{Gnarails.template_file} --skip-test-unit --database=postgresql"
+          default_options = "-m /Users/jarvis/Code/Gnar/gnarails/lib/../gnarly.rb --asset-pipeline=propshaft --skip-test-unit --css=tailwind --javascript=esbuild --database=postgresql"
```

Comment:
Looks like this path needs to be made dynamic

Reply:
Taken care of un a future commit